internal/cacerts/cacerts.go

```diff
@@ -0,0 +1,14 @@
+// Package cacerts provides an embedded CA root certificates bundle.
+package cacerts
+
+//go:generate curl -sfL https://curl.se/ca/cacert.pem -o ca-certificates.crt
```

Comment:
LazyQ: What license are these under? if it involves attribution are we correctly doing so?

Reply:
Great question. It's built off the [mozilla cert bundle](https://hg.mozilla.org/releases/mozilla-release/raw-file/default/security/nss/lib/ckfw/builtins/certdata.txt) which is under Mozilla Public License

Reply:
I _think_ this means the `cacerts` package also has to be MPL-2.0 (because we embed the MPL-2.0 data in the sources), and MPL-2.0 is (albeit weakly) copyleft.

Reply:
I'm not exactly sure what we need to do for this one.

Reply:
I think a comment on the variable that the contents are MPL-2.0 licensed might be good enough? (and having a link to the cert bundle is helpful for understanding what they are anyway)